lib/tests/2wp.js

```diff
@@ -30,6 +32,7 @@ const execute = (description, getRskHost) => {
     before(async () => {
 
       btcTxHelper = getBtcClient();
+      rskTxHelpers = getRskTransactionHelpers();
       rskTxHelper = getRskTransactionHelper(getRskHost());
```

Comment:
I left it like this because other files that will call this one can call it with this:

```
() => Runners.hosts.federates[Runners.hosts.federates.length-1].host
```

 For example: https://github.com/rsksmart/rootstock-integration-tests/blob/main/tests/04_00_04-2wp_after_fedchange.js
 
 In this case, the file `04_00_04-2wp_after_fedchange` (I know this will eventually change) does a federation change, excluding the first 2 federators and adding 2 more. This is why it sends the last federator to be the `rskTxHelper`. Because the first and second ones won't be able to do calls like `updateBridge`, etc., since they will not be part of the federation.
 
 Hardcoding the first one for all the test will conflict with this `after federation change` test. I know this might change, but right now this is what we have
 
 What do you think?

Reply:
To avoid confusion maybe, I can call the triggerRelease like this instead:

```
await triggerRelease(getRskTransactionHelpers(), btcTxHelper);
```